test/verify/check-users

```diff
@@ -61,6 +61,10 @@ def createUser(browser, user_name, real_name, password, locked, force_password_c
 @nondestructive
 @skipDistroPackage()
 class TestAccounts(MachineCase):
+    def setUp(self):
+        super().setUp()
+
+        self.sed_file("s/#KillUserProcesses=no/KillUserProcesses=yes/g", "/etc/systemd/logind.conf")
```

Comment:
Doesn't that require restarting systemd-logind.service?

But actually, if it's only about waiting for empty "Closing..." sessions to go away, then restarting logind by itself will help. See commit e1969f1027e . So if you previously saw this on test cleanup, that commit should have fixed it. If you see it *during* a test, then you need to replicate that inside the test.

Of course, if there are actually some running processes in the session, then you need both.